pkgs/development/libraries/quazip/default.nix

```diff
@@ -1,4 +1,4 @@
-{ fetchFromGitHub, lib, stdenv, zlib, qtbase, cmake, fixDarwinDylibNames }:
+{ fetchFromGitHub, lib, stdenv, zlib, qtbase, qt5compat ? null , cmake, fixDarwinDylibNames }:
```

Comment:
```suggestion
{ fetchFromGitHub, lib, stdenv, zlib, qtbase, qt5compat, cmake, fixDarwinDylibNames }:
```

Reply:
`qt5compat` contains unsupported Qt5 API's for Qt6 and only available in `qt6Packages` scope. Since we are calling this derivation from both `qt6Packages` and `libsForQt5` scope, we need to keep this.